wiki/Tournaments/OMC/OMC5.md

```diff
@@ -0,0 +1,77 @@
+# osu! Fan Cafe Mapping Contest #5 
+
+![{ABBR} Logo](https://i.imgur.com/2a7gVVl.jpg)
+
+The **OMC5** (***osu! Fan Cafe Mapping Contest #5***) is a mapping contest hosted in Korea and only Koreans are permitted to participate. It is the 5th installment of the OMC.
+
+## Contest Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-08-12 ~ 2019-08-18 24:00 UTC +9 |
+| Submitting Phase | 2019-08-19 ~ 2019-09-19 24:00 UTC +9 |
+| Judging Phase | 2019-09-20 ~ 2019-09-27 24:00 UTC +9 |
+| Results Announce | 2019-09-20 |
+
+Schedules may be delayed.
+
+## Songs
+ Contestants are offered 2 songs to choose from. These maps are free from copyright issues as we have legally bought the rights to use the songs. No other songs may be mapped other than the 2 offered.
+
+|     |
+| --: |
+| 1. (to be announced) |
+| 2. Cranky - R176 |
+
+## Ruleset
+
+### Rules
+- This contest is for only osu!Standard mode.
+- Only Koreans are permitted to participate. If your profile flag is not showing you're in korea because of some reasons like foreign residence, you can participate if you're able to verify that you're a korean somehow. 
```

Comment:
```suggestion
- Only Koreans are permitted to participate. If your profile flag is not showing that you're in Korea because of some reasons like foreign residence, you can still participate as long as you're able to verify that you are Korean.
```